src/lib/tls/sessions_sql/tls_session_manager_sql.cpp

```diff
@@ -296,6 +296,9 @@ void Session_Manager_SQL::prune_session_cache()
    {
    // internal API: assuming that the lock is held already
 
+   if(!m_max_sessions)
+      return;
```

Comment:
For clarity, maybe:

```suggestion
   if(m_max_sessions == 0)
      return;
```

Reply:
I'm not familiar with how the tests work/what exactly you want it to test, so I'll have to leave that to somebody else.